packages/contracts-bedrock/contracts/universal/FeeVault.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { L2StandardBridge } from "../L2/L2StandardBridge.sol";
+import { Predeploys } from "../libraries/Predeploys.sol";
+
+/**
+ * @title FeeVault
+ * @notice The FeeVault contract has the base logic for handling transaction fees.
+ */
+contract FeeVault {
+    /**
+     * @notice Emits each time that a withdrawal occurs
+     */
+    event Withdrawal(uint256 value, address to, address from);
+
+    /**
+     * @notice Minimum balance before a withdrawal can be triggered.
+     */
+    uint256 public immutable MIN_WITHDRAWAL_AMOUNT;
+
+    /**
+     * @notice Wallet that will receive the fees on L1.
+     */
+    address public immutable recipient;
```

Comment:
These should be consistently cased.

Reply:
Will make this all caps to follow the recent decision that constants are all caps